### DISCUSSION TITLE
Cannot edit my issue in mobile, but can in browser

### DISCUSSION THREAD

**Discussion by KalleOlaviNiemitalo** [community/community#64364] [Mobile]
When I try to edit the description of <https://github.com/dotnet/csharpstandard/issues/910>, the iOS mobile app claims I don't have permission.  I have not been granted permissions on the repository but I am the reporter of the issue.  (I am also classified as a contributor in the repository.)  I can edit the description of that issue in a web browser, and can edit my comments on the issue in the mobile app, just not the description.

I didn't have this problem before I upgraded the app to a version that supports multiple accounts.  I have never registered any other GitHub accounts in the app, though.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.126.0 (107057637)
Device: iPhone SE 2 (iOS 16.6)

![screen shot of permission error](https://github.com/community/community/assets/46201428/4bbc665f-5110-4a58-a503-511939fba1c3)
</details>

The purpose of this feedback is to help GitHub personnel notice and the fix the bug.  I can run diagnostic steps to collect more information for that purpose, but I am not interested in workarounds.

**Comment by Jaber-Saed** [comment]
It sounds like you've encountered a specific issue with editing the description of an issue on GitHub using the iOS mobile app, particularly after upgrading to a version that supports multiple accounts. While I can't provide direct troubleshooting for app-specific issues, I can offer some general suggestions that might help resolve the problem:

1. **Check Permissions:**
   - As you mentioned, you are the reporter and a contributor in the repository. However, it's always good to double-check your permissions. Make sure that your contributor status is active and that there are no restrictions that could be causing the issue.

2. **Log Out and Log In:**
   - Sometimes, logging out of the app and then logging back in can help resolve unexpected permission-related issues.

3. **Clear Cache:**
   - Clearing the app's cache might help refresh the data and resolve any cached issues causing the problem.

4. **Update the App:**
   - Ensure that you are using the latest version of the GitHub app. App updates often include bug fixes and improvements that might resolve such issues.

5. **Restart the App:**
   - Close the app completely and then reopen it. This can sometimes help resolve minor glitches.

6. **Contact GitHub Support:**
   - If the issue persists, consider reaching out to GitHub Support. They can provide direct assistance for app-related issues and help troubleshoot the problem.

7. **Report a Bug:**
   - If you suspect that the issue is related to a bug in the app, you can use the "Provide Feedback" or "Report a Bug" option within the app to report the problem to GitHub.

8. **Web Browser Alternative:**
   - If editing the description is crucial and you're not able to do so within the mobile app, you can temporarily switch to a web browser on your mobile device to make the necessary changes.

Remember that app behavior and features can change over time, and issues might be related to the app's specific implementation. For the most accurate and up-to-date assistance, consider reaching out to GitHub's official support channels or consulting the app's documentation or support resources.

**Comment by masoudmanson** [comment]
Facing the exact same issue. Have you had any luck with it? 🤔

**Reply by KalleOlaviNiemitalo** [reply]
I have not hit the same problem again.  However, I have upgraded the mobile app several times after I posted about the problem, and I have not tried to edit the description of the same issue with the newer versions of the app.

**Comment by marco12go** [comment]
Mobile
